index.html

```diff
@@ -0,0 +1,785 @@
+              <img
+                src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152646im_/https://img-cdn.tnwcdn.com/image?url=https%3A%2F%2Fpbs.twimg.com%2Fprofile_images%2F1198933202947203078%2FyGxNiAdA.png&amp;signature=d0c4ef35a8e410806bed50513f59f79b"
+              alt="#"
+              />
+              <h4>SwissdeCode</h4>
+              <div>
+                <h5>$4.2M$40M</h5>
+                <span>Venture Capital</span>
+              </div>
+            </a></div>
+            <div>
+              <ul>
+                <li><a href="#">Biotechnology</a></li>
+                <li><a href="#">VisVires New Protein</a></li>
+              </ul>
+            </div>
+            <div>
+              <p>
+                BREAKING: Food testing startup SwissDeCode nets $4.2m from VisVires New Protein<span>on AgFunderNews</span>
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="fndg-3">
+          <div class="fng-row">
+            <div class="fng-head"><a href="#">
+              <img
+                src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152646im_/https://img-cdn.tnwcdn.com/image?url=https%3A%2F%2Fpbs.twimg.com%2Fprofile_images%2F741095900181757955%2F3Y3ZeDBE.jpg&amp;signature=0469f20c9ad141290a048b6027e509cf"
+                alt="#"
+              />
+              <h4>Klook</h4>
+              <div>
+                <h5>$200M</h5>
+                <span>SERIES E</span>
+              </div>
+            </a></div>
+            <div>
+              <ul>
+                <li><a href="#">Internet</a></li>
+                <li><a href="#">Boyu Capital, +4</a></li>
+              </ul>
+            </div>
+            <div>
+              <p>
+                SoftBank-backed travel platform Klook raises $200M amid COVID-19<span>on TechCrunch</span>
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="fndg-3 fndg3-pc">
+          <div class="fng-row">
+            <div class="fng-head"><a href="#">
+              <img
+                src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152646im_/https://img-cdn.tnwcdn.com/image?url=https%3A%2F%2Fpbs.twimg.com%2Fprofile_images%2F1306194011699507200%2FsZgWOOx0.jpg&amp;signature=70b7da824adf2d8c49abb7df915c09a3" alt="#"/>
+              <h4>Wolt</h4>
+              <div>
+                <h5>$530M</h5>
+                <span>VENTURE CAPITAL</span>
+              </div>
+            </a></div>
+            <div>
+              <ul>
+                <li><a href="#">Mobile, +5</a></li>
+                <li><a href="#">DST Global, +9</a></li>
+              </ul>
+            </div>
+            <div>
+              <p>
+                Food delivery startup Wolt raises $530M after tripling revenues <span>on SiliconANGLE</span>
+              </p>
+            </div>
+          </div>
+        </div>
+    </div>
+  </section>
+</section>
+
+<!-- <section id="section4">
```

Comment:
## Remove old pieces of commented code.
- [ ] Do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history just as explained [here](https://microverse.zendesk.com/hc/en-us/articles/360057489093-General-Requirements-for-Microverse-Projects).